nncf/experimental/onnx/sampler.py

```diff
@@ -0,0 +1,60 @@
+"""
```

Comment:
This should be named "samplers.py", like you have "collectors.py" not "collector.py".

Reply:
Done